.vuepress/public/sample-template.yaml

```diff
@@ -92,4 +92,6 @@ geosearch_guide_filter_usage_2: |-
 geosearch_guide_sort_settings_1: |-
 geosearch_guide_sort_usage_1: |-
 geosearch_guide_sort_usage_2: |-
-authentication_header_1: |-
+set_primary_key_on_index_creation: |-
+set_primary_key_on_document_addition: |-
```

Comment:
As per convention with the other code samples to quickly infer where the code samples are use, should the naming not be prefixed with 
`document_guide` ? 

I do realise it makes the id long, maybe it can be shortened to 
`document_guide_index_primary_key`
`document_guide_add_document_primary_key`